src/Tools/dotnet-monitor/EndpointInfo/ScopedEndpointInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using Microsoft.Diagnostics.NETCore.Client;
+using System;
+
+namespace Microsoft.Diagnostics.Tools.Monitor
```

Comment:
So this class exists just to create a scoped service correct?

Reply:
Correct, this allows other scoped services to to just compose the only endpoint and not have to worry about how to determine which one it should consider.